### PR TITLE
DEV-8469 Added example of using collection properties

### PR DESCRIPTION
**Pull request by MikeMcGarry**

### CODE REVIEW

File: examples/use-cases/wealth-management/Households.ipynb [hunk 1]
```diff
@@ -2,15 +2,22 @@
  "cells": [
```
Comment (gderham) [inline examples/use-cases/wealth-management/Households.ipynb:2]: <p>"instrument defintions" typo</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (gderham) [inline examples/use-cases/wealth-management/Households.ipynb:2]: <p>The other_account_owner_id property value <code>NaN</code> appears to originate in the input csv data. but could this be misinterpreted as a LUSID error?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (MikeMcGarry) [reply examples/use-cases/wealth-management/Households.ipynb:2]: <p>Good catch, fixed.</p>

Comment (MikeMcGarry) [reply examples/use-cases/wealth-management/Households.ipynb:2]: <p>Good point, I've removed the printing of a property if there was no value to upsert.</p>

**Pull request merged**